arm-templates/acr.json

```diff
@@ -0,0 +1,30 @@
+{

```

Comment:
create a new folder under templates called resources and make sure that the file name is same as template name